.github/workflows/test.yaml

```diff
@@ -722,6 +724,55 @@ jobs:
         if: failure()
         run: bash .github/slack/notify-failure.sh ${{ github.job}} ${{ matrix.platform }}
 
+  data-proxy:
+    needs: start-time
+    timeout-minutes: 60 # can take longer if platforms are down, so better protect
+    strategy:
+      fail-fast: false
+      matrix:
+        platform:
+          - cloudflare-workers
+          - vercel-edge-functions
+        clientEngine: ['dataproxy']
+        os: [ubuntu-latest] #, windows-latest, macos-latest]
+    runs-on: ${{ matrix.os }}
+    concurrency: ${{ github.run_id }}-platforms-serverless-${{ matrix.platform }}
+    env:
+      START_TIME: ${{ needs.start-time.outputs.start-time }}
+      NPM_CONFIG_LOGLEVEL: error
+      NODE_ENV: development
+      NODE_MODULES_CACHE: false
+      NODE_VERBOSE: true
+      AES_256_PASS: ${{ secrets.AES_256_PASS }}
```

Comment:
Can we give this secret a more descriptive name? Pass to what?

Reply:
We ran out of secrets, so I started a new thing https://github.com/prisma/e2e-tests/pull/2141/files#diff-73675af499015e989f5ef5d2b73578045fd72da0a3da6bbdc09593e1375ace8dR5 that decrypts and exports env vars via https://github.com/prisma/e2e-tests/pull/2141/files#diff-a952e0014885757e976dc08bf8f819f60447dd452ebe828bd5684868a53d155aR5. They are encrypted and decrypted via the `AES_256_PASS` https://github.com/prisma/e2e-tests/pull/2141/files#diff-5bc97016adbf2b476d3412f84c0f12c70dfd3e4d37d41cf0327a6875b78981d5R17. I inspired from their docs to work around this 100-secret limitation, [in this section](https://docs.github.com/en/actions/security-guides/encrypted-secrets#limits-for-secrets).

Reply:
Ah that makes sense. 

I think we can actually delete a _lot_ of the existing secrets. Will make a TODO for myself to clean up the list some time soon.